src/components/UI/Display/Apps/AppDetailsNew/utils.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Can this also be a TypeScript file?

Reply:
Yes! but can you perhaps help me figure out the type for headingProps in Headingrenderer.

Reply:
`headingProps` is supposed to contain props for a `h1(2,3,4,5,6)` element, right? As well as the `level` prop.

How about this?
```typescript
interface IHeadingProps extends React.ComponentPropsWithoutRef<'h1'> {
	level: 1 | 2 | 3 | 4 | 5 | 6;
}
```